src/string_util.h

```diff
@@ -39,6 +39,10 @@ inline std::string StrCat(Args&&... args)
 void ReplaceAll(std::string* str, const std::string& from,
                 const std::string& to);
 
-} // end namespace benchmark
+std::string GenerateInstanceName(const std::string& name, int arg_count,
```

Comment:
sorry, this doesn't fit here either. a new header would be reasonable in this case, i think. a private benchmark_util.h maybe?